docs/source/guide/auth_setup.md

```diff
@@ -35,39 +36,44 @@ Set up Label Studio Enterprise as a Service Provider (SP) with your Identity Pro
-    3. Click **+ Add Mapping** to map additional workspaces to groups. 
+    - In the **Workspaces to Groups Mapping** section, click **+ Add Mapping**.
+    - In the **Workspace** field, type a new workspace name or select a workspace from the drop-down list. Then, type a **Group** that matches a group name sent as an attribute in a SAML authentication response by your IdP. You can map multiple groups to the same workspace. 
+    - Click **+ Add Mapping** to map additional workspaces to groups. 
 9. You can also map specific user groups to specific user roles in Label Studio Enterprise. 
-   1. In the **Roles to Groups Mapping** section, click **+ Add Mapping**.
-   2. Select a **Role** from the drop-down list of options, then type a **Group** that matches a group name sent as an attribute in a SAML authentication response by your IdP. You can map multiple groups to the same role. 
-   3. Click **+ Add Mapping** to map additional roles to groups. 
+   - In the **Roles to Groups Mapping** section, click **+ Add Mapping**.
+   - Select a **Role** from the drop-down list of options, then type a **Group** that matches a group name sent as an attribute in a SAML authentication response by your IdP. You can map multiple groups to the same role. 
+   - Click **+ Add Mapping** to map additional roles to groups. 
 10. Click **Save** to save your SAML and SSO settings. 
 
 Test the configuration by logging in to Label Studio Enterprise with your SSO account.
 
+
 ### Setup SAML SSO with Okta video tutorial
 
 <iframe class="video-border" width="560" height="315" src="https://www.youtube.com/embed/Dr-_hyWIw4M" width="100%" height="400vh" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+
 ## Set up LDAP authentication 
 
-After you set up LDAP authentication, you can no longer use native authentication to log in to the Label Studio UI. Set up LDAP authentication and assign LDAP users to your Label Studio Enterprise organization using environment variables in Docker. 
+!!! note
+    After you set up LDAP authentication, you can no longer use native authentication to log in to the Label Studio UI. 
+
+Set up LDAP authentication and assign LDAP users to your Label Studio Enterprise organization using environment variables in Docker. 
 
-You can also map specific LDAP groups to specific organization roles and workspaces in Label Studio Enterprise, making it easier to set up and manage role-based access control (RBAC) and project access in Label Studio Enterprise. 
+You can also map specific LDAP groups to specific organizational roles and workspaces in Label Studio Enterprise, making it easier to set up and manage role-based access control (RBAC) and project access in Label Studio Enterprise. 
```

Comment:
not sure `organizational` is the correct meaning here. There should meaning for `roles of the organization`, `organization's roles`

Reply:
Fixed.